packages/common/src/queue.ts

```diff
@@ -182,5 +236,9 @@ export const createQueue = <returnType, taskType = void>({
       }
       return emptyPromiseWithResolvers.promise;
     },
+    setParameters: (_parameters) => {
+      validateParameters(_parameters);
+      parameters = _parameters;
+    },
```

Comment:
If you initialize with both parameters defined, then call `setParameters` with only one defined, I think it will overwrite the other one here (as `undefined`). Just want to double check that's the expected behavior.

Reply:
Nice call - will make it so that it only updates named keys